components/flowerFooter/FlowerFooter.tsx

```diff
@@ -1,40 +1,28 @@
 import EmailIcon from '@mui/icons-material/Email';
 import InstagramIcon from '@mui/icons-material/Instagram';
-import {
-  Box,
-  SxProps,
-  Typography,
-  useMediaQuery,
-  useTheme
-} from '@mui/material';
-const catFooterImage = `${process.env.NEXT_PUBLIC_SUPABASE_URL}/storage/v1/object/public/considerate%20cat%20assets/catNamedShoe.png`;
+import { Box, Typography, useMediaQuery, useTheme } from '@mui/material';
 
-interface flowerFooterProps {
-  sx?: SxProps;
-}
+const catFooterImage = `${process.env.NEXT_PUBLIC_SUPABASE_URL}/storage/v1/object/public/considerate%20cat%20assets/catNamedShoe.png`;
```

Comment:
**thought:** Do you think we should optimize this image and the one for the footer background and put it in the project files?

Reply:
Yes i think it is a good idea, have we done this yet ? I thought we were pulling all the images directly from supabase.

Reply:
We have, I've been putting optimized component images in the `public` dir.

Reply:
Got it, changing this.

Also lol, did you name this or did i?
![image](https://github.com/user-attachments/assets/fce7f91e-886d-4f38-a888-418c7136574d)

Reply:
Would you put both footer images in public? How do you decide what goes in public and what comes from supabase?

Reply:
Done :)